core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -421,6 +423,7 @@ public final void runInContext() {
     @Override
     public void closed(Status status, Metadata trailers) {
       Long timeoutMicros = getRemainingTimeoutMicros(callOptions.getDeadlineNanoTime());
+      transportFuture.cancel(false);
```

Comment:
If this is false, and the transport future is already being created, the future will complete.  false just makes it so if it has not yet been started, it will never start.  

Is it going to be common that TransportFutures will be given out that have not been submitted to an executor?